CONTRIBUTING.md

```diff
@@ -0,0 +1,134 @@
+# Contributing to Miden Base
+
+#### First off, thanks for taking the time to contribute!
+
+We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a [bug](https://github.com/0xMiden/miden-base/issues/new?assignees=&labels=bug&projects=&template=1-bugreport.yml)
+- Taking part in [discussions](https://github.com/0xMiden/miden-base/discussions)
+- Submitting a [fix](https://github.com/0xMiden/miden-base/pulls)
+- Proposing new [features](https://github.com/0xMiden/miden-base/issues/new?assignees=&labels=enhancement&projects=&template=2-feature-request.yml)
```

Comment:
The title and links here are for the miden-base repo. I guess this should be changed to miden-faucet.